RUNTIME ERROR: Assertion failed. 
  != 

-------------------------------------------------
	<std>:787:7-50	function <anonymous>

      error 'Assertion failed. ' + a + ' != ' + b,

-------------------------------------------------
	testdata/assert_equal5:1:1-29	$

std.assertEqual("\n ", "\n")

-------------------------------------------------
	During evaluation	


